src/common/base/Cargo.toml

```diff
@@ -16,8 +16,10 @@ common-error.workspace = true
 common-macro.workspace = true
 futures.workspace = true
 paste = "1.0"
+pin-project.workspace = true
 serde = { version = "1.0", features = ["derive"] }
 snafu.workspace = true
+tokio.workspace = true
```

Comment:
Now we have two more dependencies (also the tokio) of the common-base 😢 

Maybe we should move the range_read mod or its implementations to object-store crate......